Jenkinsfile-datastax

```diff
@@ -27,7 +27,7 @@ def initializeEnvironment() {
   env.GITHUB_BRANCH_URL = "${GITHUB_PROJECT_URL}/tree/${env.BRANCH_NAME}"
   env.GITHUB_COMMIT_URL = "${GITHUB_PROJECT_URL}/commit/${env.GIT_COMMIT}"
 
-  env.MAVEN_HOME = "${env.HOME}/.mvn/apache-maven-3.6.3"
+  env.MAVEN_HOME = "${env.HOME}/.mvn/apache-maven-3.8.8"
```

Comment:
This won't work out of the gate; I have to update the AMI def for the runner that executes this logic to include a new Jenkins version.  This can probably stay as 3.8.8 is the current 3.8.x release as of this writing... I'm just saying it's not enough by itself.

I'll take care of adding the new version to the runner.

Reply:
PR to make this change is now up

Reply:
The corresponding PR for the DataStax Jenkins runner has been merged and I'm working on building a new runner now.  Assuming we merge this once we have an image on the ASF CI side that includes the proper version of Maven I think we'll be all set.

In short: there's no reason to take this out that I can see.